src/test/cargo/simple-main/main.expected

```diff
@@ -0,0 +1 @@
+[main.assertion.1] line 4 assertion failed: 1 == 2: FAILURE
```

Comment:
Why did you change the expected output from this test?

Reply:
I did so because I noticed that the current cargo-rmc tests do not check for the behavior of `cargo-rmc` when a test fails. I modified this test with a simple assertion to ensure that
1. `cargo-rmc` does not accept all inputs.
2. Fails for "right" reasons.